src/fetch.ts

```diff
@@ -2,17 +2,28 @@ import type { RpcGraphQL } from '@solana/rpc-graphql';
 
 export async function gql(...params: Parameters<RpcGraphQL['query']>) {
     const [source, variableValues] = params;
-    const res = await fetch('/api/graphql', {
+    return await postRequest('/api/graphql', { source, variableValues });
+}
+
+export async function _fetch(url: string, body: object) {
+    return await postRequest(url, body);
+}
```

Comment:
There's no reason for this function anymore, since the app can now just call `postRequest` directly.